lib/husky_musher/app.py

```diff
@@ -48,6 +48,7 @@ def set_cache_control(response):
 
 @app.errorhandler(404)
 def page_not_found(error):
+    app.logger.error(f'Page Not Found', exc_info=error)
```

Comment:
I'm inclined to log this at an info level. Its useful to know about, e.g. could alert us to broken links. However, if a user is entering bogus URLs, in many cases that doesn't indicate an error in the eyes of the application.

Reply:
Makes a lot of sense, will update for Monday.